### PR TITLE
Speed up package management on travis

### PR DESCRIPTION
**Pull request by szepeviktor**
/vendor should not be cached!

### CODE REVIEW

**Comment by szepeviktor** [comment]
Test need fixing, see for example #12726 

Please merge.

**Comment by szepeviktor** [comment]
@tsteur Maybe this one too?

**Comment by szepeviktor** [comment]
@Findus23 Any chance merging this?

**Comment by Findus23** [comment]
@szepeviktor Indeed. [This](https://travis-ci.org/matomo-org/matomo/jobs/369967937#L673) shows that the install currently takes about 30s, so your change will speed it up by that amount (minus the time composer needs for updating), correct?

**Comment by szepeviktor** [comment]
@Findus23 Yes, bit-by-bit

File: .travis.yml [hunk 1]
```diff
@@ -138,6 +138,7 @@ after_success:
 cache:
   directories:
     - "travis_phantomjs"
+    - $HOME/.composer/cache
```
Comment (sgiehl) [inline .travis.yml]: wouldn't it be better to only cache `$HOME/.composer/cache/files`, as everything else might change to often, causing cache invalidations?

Comment (szepeviktor) [reply .travis.yml]: Maybe you're right.
I contribute to many open-source projects - like WP-CLI - and we use this directory.

Comment (szepeviktor) [reply .travis.yml]: Change frequency equals how many times *we* change the lock file.

Comment (sgiehl) [reply .travis.yml]: Guess our lock file isn't changed that frequently

Comment (szepeviktor) [reply .travis.yml]: Yes https://github.com/matomo-org/matomo/commits/3.x-dev/composer.lock

**Pull request merged**